build.sbt

```diff
@@ -330,6 +331,29 @@ lazy val scioAvro: Project = Project(
   )
 ).configs(IntegrationTest)
 
+lazy val scioBigQueryTestPipeline: Project = Project(
+  "scio-bigquery-test-pipeline",
+  file("scio-bigquery-test-pipeline")
+).settings(
+  // necessary to properly test
```

Comment:
Nit, I would change the comment to why its necessary to properly test (need available at compile time)